pkgs/development/r-modules/cran-packages.nix

```diff
@@ -21212,7 +21213,6 @@ in with self; {
   constrainedKriging = derive2 { name="constrainedKriging"; version="0.2.4"; sha256="1a91s0b7yka37fb5pm172fmlqrhm6da370cqb9knvkg5n8vi4hys"; depends=[RandomFields rgeos sp spatialCovariance]; broken = true; };
   contact = derive2 { name="contact"; version="1.2.8"; sha256="1db3vgjlz0ivw3sx37mwy7ngjx0in3rxlm3i3akh4xzbfp9gjj87"; depends=[ape data_table doParallel foreach igraph lubridate raster rgdal rgeos sp]; broken = true; };
   contextual = derive2 { name="contextual"; version="0.9.8.4"; sha256="0lww36nymg2cla83jdvl7a42cwn0il20apciz0ddplq8rw2i0a91"; depends=[data_table doParallel foreach Formula iterators itertools R_devices R6 rjson]; broken = true; };
-  conting = derive2 { name="conting"; version="1.7"; sha256="08f271j3gxcp4shda08kkv0v9ayaadvxgcglfsn6hjmw4hn298zg"; depends=[BMS coda gtools mvtnorm tseries]; broken = true; };
```

Comment:
Conting still seems to be removed from CRAN, so unmarking the broken flag is not the solution here. What is upstream's solution to the removal of conting?

Reply:
Ah, thanks for the catch.
So if something is marked as broke in rPackages, that's usually because it's been removed?

I found that they use the vandenman/contig github repo
I'll try to use that instead

Reply:
> Ah, thanks for the catch. So if something is marked as broke in rPackages, that's usually because it's been removed?

Yes, the flag gets removed if it is restored to CRAN.

> I found that they use the vandenman/conting github repo I'll try to use that instead

:+1: